File: Data Structures & Algorithms/first-unique-character-in-a-string/submission-1.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        
        int ct = 0;
        bool is = false;
        for( int i=0; i<s.length(); i++ ) {
            is = false;
            for( int j=0; j<s.length(); j++ ) {
                if( i !=  j ) {
                if( s[i] == s[j] ) {
                    is = true;
                }
                }
            }
            if( is == false ) {
                return i;
            }
        }
        return -1;

    }
};

File: Data Structures & Algorithms/sort-an-array/submission-1.java
class Solution {

    public void swap( int[] arr , int i , int j ) {
         int c = arr[i];
         arr[i] = arr[j];
         arr[j] = c;
    }
    public int[] sortArray(int[] nums) {
        
       //int[] ans  = new int[nums.length];
        for( int i=0; i<nums.length-1; i++ ) {
            for( int j=0; j<nums.length-i-1; j++ ) {
                if( nums[j] > nums[j+1] ) {
                    swap( nums , j , j+1);
                }
            }
        }
        return nums;
    }
}